tests/unit_tests/test_unit.py

```
@@ -1911,586 +1912,130 @@ def test_spectrum_preprocessing(tmp_path, mgf_small):
-        scores[:, : step + 1, :] = torch.rand(
-            batch * beam, step + 1, vocab, device=device
-        )
-        scores[:, step, range(1, 4)] = torch.tensor(
-            [1.0, 2.0, 3.0], device=device
-        )
+def test_scan_num_in_mztab_spectra_ref(tmp_path):
+    """
+    After set_mgf_scan_index is called, MztabWriter must embed the scan
+    number in spectra_ref: 'ms_run[1]:index=0 scan=17'.
+    """
+    import pyteomics.mztab
```

Ruff (0.15.4)

[error] Redefinition of unused `pyteomics.mztab` from line 29: `pyteomics.mztab` redefined here

Remove definition: `pyteomics.mztab`

(F811)